src/main/java/com/yahoo/bullet/rest/service/QueryService.java

```diff
@@ -43,21 +43,21 @@
      * Constructor that takes various necessary components.
      *
      * @param storageManager The non-null {@link StorageManager} to use.
-     * @param responders The non-empty {@link List} of {@link PubSubResponder} to use.
+     * @param pubSubResponderList The non-null {@link PubSubResponderList} to use.
      * @param publishers The non-empty {@link List} of {@link Publisher} to use.
      * @param subscribers The non-empty {@link List} of {@link Subscriber} to use.
      * @param sleep The time to sleep between checking for messages from the pubsub.
      */
     @Autowired
```

Comment:
Instead of Autowiring this and having to expose PubSubResponders outside of PubSubConfiguration, can we create the QueryService bean there using this constructor but taking a List<PubSubResponder>?